build-tools/packages/build-cli/README.md

```diff
@@ -459,8 +459,7 @@ USAGE
 
 FLAGS
   -v, --verbose      Verbose logging.
-  --dirname=<value>  [default:
-                     C:\Users\sdeshpande\Documents\FluidFramework\build-tools\packages\build-cli\dist\commands\run]
+  --dirname=<value>  [default: /home/tylerbu/code/FluidFramework/build-tools/packages/build-cli/dist/commands/run]
```

Comment:
@sonalivdeshpande I think we need to change this to not use the oclif default property. I opened a bug.